src/arrayMethodSort.js

```diff
@@ -4,8 +4,23 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 0; j < this.length - 1; j++) {
+        if (
+          compareFunction
```

Comment:
For cleaner code, you may use default argument here:

```suggestion
const defaultCompare = (a, b) => a == b ? 0 :
  a > b ? 1 : 0

  [].__proto__.sort2 = function (compareFunction = defaultCompare) {
    for (let i = 0; i < this.length; i++) {
      for (let j = 0; j < this.length - 1; j++) {
        if ( compareFunction(this[j], this[j + 1]) > 0)
          ...
```